setup.py

```diff
@@ -69,7 +69,7 @@
 ext = '.pyx' if USE_CYTHON else '.cpp'
 
 extra_compile_args = {
-    'msvc': ['/std:c11', '/EHsc'],
+    'msvc': ['/EHsc'],
```

Comment:
Just to make sure a recent enough version of MSVC is used, I would keep `/std:c++14`.